repos/spack_repo/builtin/packages/py_evaluate/package.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack_repo.builtin.build_systems.python import PythonPackage
+
+from spack.package import *
+
+
+class PyEvaluate(PythonPackage):
+    """Evaluate is a library that makes evaluating and comparing models and
+    reporting their performance easier and more standardized."""
+
+    homepage = "https://github.com/huggingface/evaluate"
+    pypi = "evaluate/evaluate-0.4.0.tar.gz"
+
+    license("apache-2.0")
+
+    version("0.4.6", sha256="e07036ca12b3c24331f83ab787f21cc2dbf3631813a1631e63e40897c69a3f21")
+    version("0.4.0", sha256="bd6a59879be9ae13b681684e56ae3e6ea657073c4413b30335e9efa9856e4f44")
+
+    depends_on("py-setuptools", type="build")
+    depends_on("python@3.7:", type=("build", "run"))
```

Comment:
According to `setup.py`, the minimum python version is `3.8`, not `3.7` for `0.4.6` (https://github.com/huggingface/evaluate/blob/a4fcb45252709bd07fba897afd4575735716664f/setup.py#L143).